README.md

```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use Theme
+To use the theme clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. To adjust the permissions use `chmod -R www-data:www-data theme-example`.
+
+If you change the name of the app from `theme-example` to another name, be sure to change the name in the file `appinfo\info.xml` as well. The folder name and `<id>' must be identical.
+
+### Edit images
+To edit images it is best to open the original images from the 'core\img' folder with Gimp or Photoshop. After you have made changes, overwrite the originals.
+
+### Change values
+Please have a look at the file `defaults.php` if you want to change texts. The copyright text on the start page and other texts are well documented there.
+
+### Change design values
+In the file `styles.css` you will find the adjustments we have preconfigured for you. You can change everything you want there. More classes and their values can be found easily with the browser console (F12 key). There you can also test directly in the browser how the change of values will be reflected. If it fits you can transfer the values to styles.css.
+
+### Integrity check
+Since files in the theme must be changeable, it has not been signed by us. Therefore an 'integrity check warning' appears in your ownCloud. You can deactivate it specifically for the theme:
```

Comment:
`appears in your ownCloud` --> `appears in your ownCloud instance`

Reply:
@mmattel Thanks Martin, I'll take your suggestions.